video/viewport.h

```diff
@@ -68,6 +68,17 @@ Point scale(Point p) {
 	return scale(p.X, p.Y);
 }
 
+// Convert to currently active coordinate system
+//
+Point toCurrentCoordinates(int16_t X, int16_t Y) {
```

Comment:
this is used to get OS coordinates from screen coordinates

a mouse position will be reported in whichever coordinate system is active.  by default that means OS coordinates, but if you've swapped to using native screen pixel coordinates you'll get pixel coordinates instead.

Reply:
Great job on this! I was wondering what the mouse coordinates would be, given that there are effectively three different methods in use on the Agon (Graphics (OS) coordinates, Text coordinates and native screen pixel coordinates). Glad to see that mouse support doesn't introduce another :-)

Reply:
this reminds me that I did want to add support for OS coordinates to bitmaps - probably by just adding in the relevant PLOT code support (which is currently missing).  supporting them for sprites will be a teensy bit more involved/complicated.  I believe that's the only graphics stuff that doesn't already support both coordinate systems